app/services/user.js

```diff
@@ -2,6 +2,33 @@ const { conflictError, databaseError } = require('../errors');
 const userModel = require('../models').user;
 const logger = require('.././logger');
 
+const calculateTotalPages = (count, limit) => {
+  if (count === 0) {
+    return 1;
+  }
+  return Math.ceil(count / limit);
+};
+
+exports.findAllPagination = paginationObject =>
+  userModel
+    .findAndCountAll({
+      limit: paginationObject.limit,
+      offset: paginationObject.offset,
+      order: [[paginationObject.orderBy, paginationObject.order]]
+    })
+    .then(users => ({
+      users,
+      totalPages: calculateTotalPages(users.count, paginationObject.limite),
+      limit: Number(paginationObject.limit),
```

Comment:
Use parseInt instead of Number, anyway this should be parsed by controller or anyone that use this service